OracleHTTPServer/README.md

```diff
@@ -1,82 +1,47 @@
+This project offers Dockerfile for Oracle HTTP Server 12.2.1.2.0 in standalone mode. To assist in building the images, you can use the buildDockerImage.sh script. See below for instructions and usage
 
 The **buildDockerImage.sh** script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call docker build with their preferred set of parameters.
 
-### Building OHS Docker Install Images
-IMPORTANT: You have to download the binaries of OHS and Oracle JDK and put them in place (see .download files inside dockerfiles/).
-
-Download the required packages (see .download files) and drop them in the folder of your distribution version of choice. Then go into the **dockerfiles** folder and run the **buildDockerImage.sh** script as root.
-
-    $ sh buildDockerImage.sh -v 12.2.1
-
-IMPORTANT: the resulting images will NOT have a domain pre-configured.
-You must extend the image with your own Dockerfile, and create your domain using WLST. You might take a look at the use case samples as well below.
-
-## Samples for OHS Domain Creation
-To give users an idea on how to create a domain from a custom Dockerfile to extend the OHS image, we provide a few samples. For an example on 12.2.1, you can use the sample inside **samples/1221-OHS-domain** folder for creating standalone OHS domain
-
-### Sample Domain for OHS 12.2.1
-This Dockerfile will create an image by extending **oracle/ohs:12.2.1-sa** . It will configure a ohsDomain with the following settings:
-
- - Oracle Linux Username: oracle
- - Oracle Linux Password: welcome1
- - OHS Domain Name: ohsDomain
- - OHS Component name : ohs_sa1
- - NodeManager on port: 5556
- - OHS on port: 7777
-
-### Write your own OHS domain with WLST
-The best way to create your own, or extend domains is by using WebLogic Scripting Tool. You can find an example of a WLST script to create domains at create-sa-ohs-domain.py. You may want to tune this script with your own setup . You can also extend images and override an existing domain, or create a new one with WLST.
-
-## Building a sample Docker Image of a OHS Domain
-To try a sample of a OHS standalone image with a domain configured, follow the steps below:
-
-Make sure you have **oracle/ohs:12.2.1-sa** image built. If not go into **dockerfiles/12.2.1** and call:
-
-        $ sh buildDockerImage.sh -v 12.2.1
+### Building Oracle JDK (Server JRE) base image
+You must first download the Oracle Server JRE binary and drop in folder `../OracleJava/java-8` and build that image. For more information, visit the [OracleJava](../OracleJava) folder's [README](../OracleJava/README.md) file.
 
-### How to Build OHS domain Image
-Go to folder **samples/1221-OHS-domain**
+        $ cd ../OracleJava/java-8
+        $ sh build.sh
 
-Run the following command:
+### Building OHS Docker Image
+IMPORTANT: You have to download the OHS binary and put it in place (see .download files inside dockerfiles/).
```

Comment:
I would specify the URL and OHS binary to download instead of relying on the contents of the `.download` file. We've had users in the past who've not completely understood this concept, so the more documentation on what to download and from where is better.